tests/Http/HttpClientTest.php

```diff
@@ -21,7 +26,7 @@
 class HttpClientTest extends TestCase
 {
     /**
-     * @var \Illuminate\Http\Client\Factory
+     * @var Factory
```

Comment:
Revert this change

Reply:
Sorry Dries. Not a clue why they changed, must have been my IDE

Reply:
@lukeraymonddowning no worries :)